spec/rspec/matchers/built_in/raise_error_spec.rb

```diff
@@ -290,9 +290,22 @@ def to_s
   end
 end
 
-RSpec.describe "expect { ... }.not_to raise_error(message)" do
+RSpec.describe "expect { ... }.not_to raise_error('message')" do
   it "issues a warning" do
-    expect_warning_with_call_site __FILE__, __LINE__+1, /risks false positives/
+    expect_warning_with_call_site __FILE__, __LINE__+1, /not_to raise_error\(message\)` risks false positives/
+    expect { raise 'blarg' }.not_to raise_error("blah")
+  end
+
+  it "can supresses the warning when configured to do so", :warn_about_potential_false_positives do
+    RSpec::Expectations.configuration.warn_about_potential_false_positives = false
```

Comment:
We have an helper ‘with_isolated_config’. If you think it’s overkill, or suspect that the example couldn’t be considered as balanced with an opposite example that doesn’t use it, I think it’s fine to restore it after ‘expect’.

Reply:
Ah, it's probably more accurate to explicitly enable it in the other test, I agree (if I'm following you?)

Reply:
(updated the existing test I copied from at the same time)